src/utils/string.js

```diff
@@ -12,4 +11,10 @@ function startsWith(str, searchStr, position) {
   }
 }
 
-export { camelize, startsWith };
+function camelToSnake(camelCaseString) {
```

Comment:
I think we should use Lodash functions, with tree shaking importing single functions is minimal on the payload size, on the other hand it is a huge boost for safety and readability, and sometimes even for performance.

I would also use arrow functions here for declaring these functions. If you are also implementing it yourself i would introduce unit tests, btw do we have any unit tests for the VP?

Reply:
Added Lodash